cmd/repo/setup/runner.go

```diff
@@ -0,0 +1,100 @@
+package setup
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/google/go-github/v44/github"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/devctl/pkg/githubclient"
+)
+
+type runner struct {
+	flag   *flag
+	logger micrologger.Logger
+	stdout io.Writer
+	stderr io.Writer
+}
+
+func (r *runner) Run(cmd *cobra.Command, args []string) error {
+	ctx := context.Background()
+
+	err := r.flag.Validate()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = r.run(ctx, cmd, args)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) error {
+	s := strings.Split(args[0], "/")
+	if len(s) != 2 {
+		return microerror.Maskf(invalidArgError, "expected owner/repo, got %s", args[0])
+	}
+
+	owner := s[0]
+	repo := s[1]
+
+	token := os.Getenv("GITHUB_TOKEN")
```

Comment:
Some validation would be useful. Or it's already validated elsewhere?

Reply:
Changed that to use `os.LookUp` and fail with `environement variable not found` when the variable does not exists.